x-pack/test/observability_functional/apps/observability/alerts/rules_page.ts

```diff
@@ -0,0 +1,176 @@
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+export default ({ getService }: FtrProviderContext) => {
+  const esArchiver = getService('esArchiver');
+  const testSubjects = getService('testSubjects');
+  const supertest = getService('supertest');
+  const find = getService('find');
+  const retry = getService('retry');
+  const RULE_ENDPOINT = '/api/alerting/rule';
+
+  async function createRule(rule: any): Promise<string> {
+    const ruleResponse = await supertest.post(RULE_ENDPOINT).set('kbn-xsrf', 'foo').send(rule);
+    expect(ruleResponse.status).to.eql(200);
+    return ruleResponse.body.id;
+  }
+  async function deleteRuleById(ruleId: string) {
+    const ruleResponse = await supertest
+      .delete(`${RULE_ENDPOINT}/${ruleId}`)
+      .set('kbn-xsrf', 'foo');
+    expect(ruleResponse.status).to.eql(204);
+    return true;
+  }
+
+  const getRulesList = async (tableRows: any[]) => {
+    const rows = [];
+    for (const euiTableRow of tableRows) {
+      const $ = await euiTableRow.parseDomContent();
+      rows.push({
+        name: $.findTestSubjects('rulesTableCell-name').find('a').text(),
+        enabled: $.findTestSubjects('rulesTableCell-ContextStatus').find('button').attr('title'),
+      });
+    }
+    return rows;
+  };
+
+  describe('Observability Rules page', function () {
+    this.tags('includeFirefox');
+
+    const observability = getService('observability');
+
+    before(async () => {
+      await esArchiver.load('x-pack/test/functional/es_archives/observability/alerts');
+      await esArchiver.load('x-pack/test/functional/es_archives/infra/metrics_and_logs');
+      await observability.alerts.common.navigateWithoutFilter();
+    });
+
+    after(async () => {
+      await esArchiver.unload('x-pack/test/functional/es_archives/observability/alerts');
+      await esArchiver.unload('x-pack/test/functional/es_archives/infra/metrics_and_logs');
+    });
+
+    describe('Feature flag', () => {
+      // Related to the config inside x-pack/test/observability_functional/with_rac_write.config.ts
+      it('Link point to O11y Rules pages by default or when "xpack.observability.unsafe.rules.enabled: true"', async () => {
+        const manageRulesPageHref = await observability.alerts.rulesPage.getManageRulesPageHref();
+        expect(new URL(manageRulesPageHref).pathname).equal('/app/observability/alerts/rules');
+      });
+    });
+
+    describe('Create rule button', () => {
+      it('Show Create Rule flyout when Create Rule button is clicked', async () => {
+        await observability.alerts.common.navigateToRulesPage();
+        await retry.waitFor(
+          'Create Rule button is visible',
+          async () => await testSubjects.exists('createRuleButton')
+        );
+        await observability.alerts.rulesPage.clickCreateRuleButton();
+        await retry.waitFor(
+          'Create Rule flyout is visible',
+          async () => await testSubjects.exists('addRuleFlyout')
```

Comment:
Is there a reason you can't use an existing test subject, like `addRuleFlyoutTitle`?